packages/theming/theming-utils/package.json

```diff
@@ -30,8 +30,8 @@
     "@fluentui-react-native/tokens": ">=0.20.14 <1.0.0"
   },
   "devDependencies": {
-    "@fluentui-react-native/design-tokens-win32": "^0.42.0",
-    "@fluentui-react-native/design-tokens-windows": "^0.42.0",
+    "@fluentui-react-native/design-tokens-win32": "^0.48.0",
+    "@fluentui-react-native/design-tokens-windows": "^0.48.0",
```

Comment:
Update title to 48?

Reply:
0.48.0 didn't fix the issues with the strings unfortunately :/ working on that now and will update the title once we have a version that works